azure_li_services/runtime_config.py

```diff
@@ -58,9 +58,9 @@ class RuntimeConfig(object):
             path: expected_unix_device_node
 
         credentials:
-          username: user_name
-          password: base64_encoded_password
-          ssh-key:  ssh_rsa_key
+          username: user
+          shadow_hash: "sha-512-cipher"
```

Comment:
Hmm I think we should accept any of the hashes supported in the shadow file. While I agree that some hashes are no longer considered secure and we want to encourage best security practices, I do not think we want to force the issue. Meaning the description should probably be "password-hash-sha-512-preferred"